charts/argo-cd/templates/argocd-configs/cluster-secrets.yaml

```diff
@@ -0,0 +1,23 @@
+{{- range .Values.configs.clusterCredentials }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "argo-cd.name" $ }}-cluster-{{ .name }}
+  labels:
+    {{- include "argo-cd.labels" (dict "context" $) | nindent 4 }}
+    argocd.argoproj.io/secret-type: cluster
+  {{- with .annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+type: Opaque
+data:
```

Comment:
Since the value file has raw strings for these, shouldn't this be stringData?

Reply:
You are right :) thx for noticing that!
Already pushed a fix.

Reply:
Yay, glad I didn't miss something obvious :)